index.js

```diff
@@ -0,0 +1,3 @@
+import './modules/manipulateClasses.js';
+import './modules/main.js';
+import './modules/luxon.js';
```

Comment:
- [ ] this is not the right way of importing using ES6, in ES6 you only import the functions/modules/classes you exported which will look something like this:
```import Book from './modules/main.js'```